ui/StatusQ/src/StatusQ/Controls/StatusButton.qml

```diff
@@ -1,24 +1,58 @@
-import QtQuick 2.14
+import QtQuick 2.15
+
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 
 StatusBaseButton {
-    id: statusButton
+    normalColor: {
+        switch(type) {
+        case StatusBaseButton.Type.Primary:
+            return Theme.palette.primaryColor1
+        case StatusBaseButton.Type.Warning:
+            return Theme.palette.warningColor3
+        case StatusBaseButton.Type.Danger:
+            return Theme.palette.dangerColor3
+        case StatusBaseButton.Type.Success:
+            return Theme.palette.successColor2
+        case StatusBaseButton.Type.Normal:
```

Comment:
```suggestion
```
Can be removed, here and in other places

Reply:
Right, my C++ brain always sees the potential `-Wswitch` warnings :laughing:

Reply:
Removed